packages/components/src/local/organisms/setting-forces/settings-force-roles/index.tsx

```diff
@@ -134,7 +134,7 @@ export const RolesAccordion: FC<PropTypes> = ({ data, handleChangeForce, forces,
           <div className={cx(styles.col, styles.section)}>
             <FormGroup placeholder="Roles">
               <SortableList
-                remove={false}
+                remove={true}
```

Comment:
@lilitkarapetyan - I've just been investigating, but I don't understand what this `remove` prop is doing.  Could you tell me please?  Then I'll add a javadoc to the props.

Reply:
This boolean prop for the visibility of the delete button

Reply:
But there is some strange `removeLocal` code, too: 
![image](https://user-images.githubusercontent.com/1108513/158568701-5ed78cc1-6beb-410c-a291-201b90f51461.png)

IT _looks-like_ if `remove` is missing, then we still show the `trash` icon.

Reply:
It seems to me that this part of the code is not needed because it should always be true, we see the icon when we hover over the role with the mouse